### ISSUE TITLE
Docs link for graphmath 1.0.2 appears to be broken.

### ISSUE THREAD

**Issue by crertel** [hexpm/hex_web#259]
Despite publishing and running hex docs, it appears that the site doesn't have the docs.

Should I rerun it, or what should I do?

**Comment by wojtekmach** [comment]
@crertel can you successfully generate docs locally with `mix docs`? I tried that on https://github.com/crertel/graphmath master branch and it gave me back an essentially blank `index.html` that redirects to `overview.html`, which doesn't exist, which is consitent with what hexdocs.pm currently serves. I also saw this output:

```
~/graphmath[master]% mix docs
Closing unclosed backquotes ` at end of input
Closing unclosed backquotes ` at end of input
Docs successfully generated.
View them at "doc/index.html"
```

which if I remember correctly points to a well known problem with earmark when parsing code snippets.

**Comment by ericmj** [comment]
@crertel Can you show the output of `mix hex.docs`? Also include the version of (`mix hex.info`) and the version of ex_doc (found in the mix.lock file).

**Comment by crertel** [comment]
Will get back to you folks tomorrow...thanks!

**Comment by ericmj** [comment]
Ping?

**Comment by crertel** [comment]
Hey, sorry for delay!

I tried running `mix hex.docs`, get the output is:

```
crertel@crockbox:~/projects/graphmath$ mix hex.docs
Closing unclosed backquotes ` at end of input
Closing unclosed backquotes ` at end of input
Docs successfully generated.
View them at "doc/index.html".
[#########################] 100%
Published docs for graphmath 1.0.2
Hosted at http://hexdocs.pm/graphmath/1.0.2/
```

Running `mix docs` yields:
```
crertel@crockbox:~/projects/graphmath$ mix docs
Closing unclosed backquotes ` at end of input
Closing unclosed backquotes ` at end of input
Docs successfully generated.
View them at "doc/index.html".
crertel@crockbox:~/projects/graphmath$ 
```

Running `mix hex.info` yields:
```
crertel@crockbox:~/projects/graphmath$ mix hex.info
Hex:    0.10.4
Elixir: 1.2.1
OTP:    18.2.1

Built with: Elixir 1.2.1 and OTP 18.2.2

A new Hex version is available (0.11.2), please update with `mix local.hex`
Registry file available (last updated: 2016-03-13 03:54:22)
Size: 1650kB (compressed 536kb)
Packages #: 1702
Versions #: 7645
crertel@crockbox:~/projects/graphmath$ 
```

**Comment by ericmj** [comment]
Sounds like this is an ex_doc issue. Hex simply calls ex_doc, zips the docs and uploads them.

You may try if it works when you fix the warnings, otherwise open an issue on ex_doc.
